dotnet-interactive.IntegrationTests/Utility/ProcessHelper.cs

```diff
@@ -61,5 +61,18 @@ public static class ProcessHelper
 
             return process;
         }
+
+        public static Process StartInteractiveServer(
+            string kernelName,
+            Action<string> output = null,
+            Action<string> error = null)
+        {
+            return Start(
+                command: "dotnet",
```

Comment:
Technically the command is `dotnet-interactive`.

Reply:
This was intentional; most user invocations will be of the form `dotnet interactive ...` which I wanted to mimic.